utils/desyscall/README.md

```diff
@@ -0,0 +1,3 @@
+# libdesyscall
+
+This library emulates in userspace various costly syscall to allow targets to scale better over cores in Linux
```

Comment:
"This Linux library emulates various costly syscalls in userspace, to allow targets to scale better over multiple cores"